### DISCUSSION TITLE
December 2023 - Community Pool & Inflation Proposal

### DISCUSSION THREAD

**Discussion by lechenghiskhan** [akash-network/community#424] [Governance Proposals]
## Timeline

Estimated GH discussion post date: December 17, 2023 
Estimated on-chain submission date: December 20, 2023
Estimated go-live date: December 27, 2023 

## Background and Context

The last inflation and community pool update was completed with [Proposal 211](https://www.mintscan.io/akash/proposals/211) at the end of July. 

As 2023 draws to a close, the Akash GPU marketplace has amassed over 160 GPUs, is running a full stack of AI workloads from inference to training, and has grown to become the most advanced and capable Supercloud. 

Now, the work of scaling the network begins — and it starts with incentives.

## Proposal

Now that scaling the network has become the foremost challenge driven by an influx of use cases and potential tenants, we are proposing to increase the community pool tax to ensure adequate funding to power provider and tenant incentives as well as the growing number of community initiatives that make SDXL on Akash and Akash Chat possible. The estimated calculations can be viewed [here](https://docs.google.com/spreadsheets/d/19OoCmh1WGKlCTvp1u63-S-8ckiMQp2DKIJrvMwNlHuo/edit#gid=1773715512). 

1. Increasing the community pool tax from 25% to 40% to grow the community pool and ensure it is properly funded for provider/tenant incentives and other community initiatives.  

2. Increase inflation from 8% to 13% to increase effective inflation and offset the economic impact on validators and stakers created by the increased tax. This will maintain incentives for securing the network. Please note that the actual impact to the effective inflation and staking APY are purely estimates and will depend on network-wide staking behavior. 

The use of community pool proceeds is and always will be requested via governance proposal submissions and determined by detailed discussions within sig-economics, steering committee, and associated working group meetings. 

Please stay tuned for a discussion about provider incentives that will be submitted soon after this proposal. 

Happy Holidays and Happy New Year!

**Comment by piber-dev** [comment]
hello @lechenghiskhan thank you for publishing this proposal draft

i am interested in hearing why this is relevant right now, is the community pool currently losing value from approved proposals? or are you simply interested in growing this to make room for future proposals which you expect to use a larger portion of this? of course there are recent price increases that play a major role here, but the community pool is at $7.2mm at the time of writing this. to make this into something we can all understand: this is enough to support the scaled support proposal now with 14 vanguards, $5k/month bounties, the events proposal, and the hackathon proposal for over 6 years without the community pool growing at all for the next 6 years. 

PS: the estimated calculations document for SDXL on akash needs to be made public if you want to reference it. this alone might answer some of my questions.

**Comment by lechenghiskhan** [comment]
Thanks for your question here @piber-dev

Though the community pool is well funded now, the upcoming year will ask a lot more of the community pool than we have asked to date. 

1. Both provider and tenant incentives, for example, will likely ask for several million over 2024 to bootstrap the supply needed to truly scale. Some numbers here to illustrate: A100 80gb (best combination of performance and cost currently and most sought after) costs about $1/hr, so incentivizing 1 for 90 days would cost 1 x 24 x 90 = $2,160. An exercise like thumper used 16-32x cluster, so we can easily see how just 10 of these initiatives could cost nearly $700k over just 90 days. This is a oversimplified example, or course, and actual usage, demand, etc will vary, but the sense of scale quickly emerges

2. Go big or go home. At this point, Akash Network has grown organically and measured because it gave us the room to bring critical features online, address unforeseen questions, and assess demand/supply forces. With most of these challenges addressed, we have two choices: launch bite sized incentives or large ones. Small incentives won't achieve the scale needed to attract the next 10,000 tenants and we risk losing momentum we so painstakingly built over the last year. 

3. We firmly believe: now is the time. We are the only fully functional Supercloud -- 2024 is the year to cement our position and continue to lead. 

4. All expenditures will be carefully thought through, posted for discussion, and reported. This is why our first few pilots will manually operated and managed to mitigate unintended issues. 

5. We don't want to get into a position where community funds are needed, but the pool is tapped out! Better to have it ready and not need it back not having it and really needing it. 

I hope this helps clear things up!
The doc has also been shared!

**Reply by piber-dev** [reply]
hello @lechenghiskhan i really like this response,

i am in favor of this proposal, i think that increasing inflation has no negative effects when the network is this large already.

i have no comment on the long-term effects on the changed fee percentage, this is too speculative to go into IMO.

i really like these initiatives and hope that many of these points will pass future proposals.

**Comment by KamuelBob** [comment]
Thanks @lechenghiskhan for putting this together.
My only comment is if you could confirm please that this doesn't impact the max supply (as Greg has confirmed separately) and hence what rough impact this may have on the date we expect to reach max supply (although that would be dependent on future decisions)

Also, the calculation sheet suggests an updated "Inflation max" of 20% but the on-chain proposal does not make this change. What is the reason for this decision?

**Reply by KamuelBob** [reply]
On deeper analysis of the calculation sheet, your use of the New Inflation as 11.82% is not accurate as with a minimum inflation set to 13%, the inflation would jump immediately to 13% if this proposal was to pass. Then given that we are currently sitting at 61.6% bonded ratio (below 67% goal), the inflation is going to increase throughout the year anyway (much as it is already doing). I believe the error is too large to simplify the calculations and not include this mechanism so it would be good if you could update the calculations please.

Then, as the document needs updating anyway, could you please update the numbers (e.g. current inflation and current supply) which have changed since the document was created.

**Reply by KamuelBob** [reply]
https://docs.google.com/spreadsheets/d/11lMmcQr5rOgrezs7GSl5oqkR01MkP6LwR8WyPS_MG5Y/edit?usp=sharing

**Reply by lechenghiskhan** [reply]
That's a great catch and not having the max inflation in the proposal was an error. We will get a new prop up correcting ASAP. Likely first thing in the new year. 

Since the current inflation isn't yet close to the current 15% max, there will be ample time to make that adjustment as the rate of inflation change (seldom discussed parameter) caps the speed at which inflation can rise or fall within a year. The bonded ratio being below the target ratio also means inflation will nudge upward as it already is, so stakers/validators will be yielding more as a result.

Separately, there's no impact to max supply as Greg indicated. The road towards max can be non linear and it's definitely the case here. 

Thank you for being so diligent and on it!

**Comment by hewi333** [comment]
@KamuelBob [Proposal 241](https://www.mintscan.io/akash/proposals/241) is live to add the missing InflationMax parameter you mentioned. Thank you as always for your diligence.

**Comment by SophoclesSummer** [comment]
This proposal was approved on 1/7/2023 by a 96% majority.
https://www.mintscan.io/akash/proposals/241
